geolocator_apple/apple/Classes/Handlers/GeolocationHandler.h

```diff
@@ -24,7 +24,8 @@ typedef void (^GeolocatorResult)(CLLocation *_Nullable location);
 
 - (void)startListeningWithDesiredAccuracy:(CLLocationAccuracy)desiredAccuracy
                            distanceFilter:(CLLocationDistance)distanceFilter
-        pauseLocationUpdatesAutomatically:(BOOL)pauseLocationUpdatesAutomatically
+                            pauseLocationUpdatesAutomatically:(BOOL)pauseLocationUpdatesAutomatically
+                            showBackgroundLocationIndicator:(BOOL)showBackgroundLocationIndicator
```

Comment:
In Objective-C it is common to align parameters on the colon `:`, please align the parameters here.
```suggestion
        pauseLocationUpdatesAutomatically:(BOOL)pauseLocationUpdatesAutomatically
          showBackgroundLocationIndicator:(BOOL)showBackgroundLocationIndicator
```